shared/trainer/defaults.yaml

```diff
@@ -1,4 +1,8 @@
 output_path: /network/tmp1/schmidtv/yb_runs/test_v1
+load_paths: 
+  p: none # none if using output_path
```

Comment:
so we can't load from one place and save in another output_path ?

Reply:
Currently not supported ;) we can make an issue if this becomes something we want to do